src/app/clusters/occupancy-sensor-server/occupancy-sensor-server.cpp

```diff
@@ -16,12 +16,156 @@
  */
 
 #include "occupancy-sensor-server.h"
+#include "occupancy-hal.h"
 
-#include <app-common/zap-generated/attributes/Accessors.h>
+#include <app/AttributeAccessInterfaceRegistry.h>
+#include <app/EventLogging.h>
+#include <app/data-model/Encode.h>
+#include <app/reporting/reporting.h>
+#include <app/util/attribute-storage.h>
+#include <lib/core/CHIPError.h>
 
-#include "occupancy-hal.h"
+using chip::Protocols::InteractionModel::Status;
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace OccupancySensing {
+
+Structs::HoldTimeLimitsStruct::Type
+    mHoldTimeLimitsStructs[MATTER_DM_OCCUPANCY_SENSING_CLUSTER_SERVER_ENDPOINT_COUNT + CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT];
+
+uint16_t mHoldTime[MATTER_DM_OCCUPANCY_SENSING_CLUSTER_SERVER_ENDPOINT_COUNT + CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT];
```

Comment:
Should these be in an anonymous namespace so they're not global variables that anything can touch from anywhere?

And the "m" prefix is for class members, which these are not.  "s" would be better if these are file statics (as they should be).

Reply:
Thanks, yes, it would be better to include them in an anonymous namespace, code added, names changed as well